modules/odc-deploying-java-applications.adoc

```diff
@@ -24,11 +24,11 @@ Use the following instructions to upload a JAR file in the *Topology* view to de
 
 . In the optional *Application Name* field, enter a unique name for your application for the resource labelling.
 
-. In the *Name* field, enter a unique component name to name the associated resources.
+. In the *Name* field, enter a unique component to name the associated resources.
```

Comment:
```suggestion
. In the *Name* field, enter a unique component name for the associated resources.
```
This phrasing does not make sense to me because it's saying that the "component" rather than the user is naming resources, and I don't think this meaning is what you're trying to express. Please use my suggested phrasing or some other alternative phrasing to get across the idea that the user is assigning a component name to a resources.

Reply:
Good catch. Thank you!